src/Supportive/Console/Application.php

```diff
@@ -62,14 +63,12 @@ protected function getDefaultInputDefinition(): InputDefinition
                 null,
                 InputOption::VALUE_REQUIRED,
                 'Location where cache file will be stored',
-                null
             ),
             new InputOption(
                 '--config-file',
                 '-c',
                 InputOption::VALUE_REQUIRED,
-                'Location of Depfile containing the configuration',
-                getcwd().DIRECTORY_SEPARATOR.'deptrac.yaml'
```

Comment:
Are we getting rid of the default?

Reply:
No not really. The config file resolution should happen now in the new $config assignment block.

I would like to get rid of the "-c|--condig-file" if there is a deptrac config with a default name.

The priority for which config to load should be in following order:

1. User provided path
2. Deptrac.php
3. Deptrac.yaml

I'm not done yet, but that's the idea behind those changes. 

Setting the default path in the "application.php" still feels wrong to me, I think it should happen where we try to parse the config.

I'll work on this one next week a bit more.